### ISSUE TITLE
[BUG] SfSellect accessibility violations

### ISSUE THREAD

**Issue by qiqqq** [DivanteLtd/storefront-ui#1213]
**Describe the bug**
There are two accessibility violations in SfSelect component:

**How to reproduce**
1. Go to https://storybook.storefrontui.io/?path=/story/molecules-select--common
2. Check "Accessibility" tab

**Expected behavior**
These violations shouldn't appear

**Screenshots**
https://prnt.sc/sn7br8
